lib/controllers/src/ActuatorController.cpp

```diff
@@ -217,4 +219,133 @@ bool ActuatorController::isPastMinAngle() {
+  isVelocityPID ? m_actuatorConfig.velocityPID.I : m_actuatorConfig.positionPID.I = newI;
+}
+
+void ActuatorController::updateActuatorConfigD(const float newD, const bool isVelocityPID) {
+  isVelocityPID ? m_actuatorConfig.velocityPID.D : m_actuatorConfig.positionPID.D = newD;
+}
+
+void ActuatorController::updateActuatorConfigBias(const float newBias, const bool isVelocityPID) {
+  isVelocityPID ? m_actuatorConfig.velocityPID.bias : m_actuatorConfig.positionPID.bias = newBias;
+}
+
+void ActuatorController::updateActuatorConfigDeadzone(const float newDeadzone, const bool isVelocityPID) {
+  isVelocityPID ? m_actuatorConfig.velocityPID.deadZoneError : m_actuatorConfig.positionPID.deadZoneError = newDeadzone;
+}
+
+// Update PIDController upon reception of new PID values
+void ActuatorController::updatePIDControllerP(const float newP, const bool isVelocityPID) {
+  // Change P, but keep I and D the same
+  isVelocityPID ? m_velocityPIDController.setTunings(newP, m_velocityPIDController.getIParam(),
+                                                     m_velocityPIDController.getDParam())
+                : m_positionPIDController.setTunings(newP, m_positionPIDController.getIParam(),
+                                                     m_positionPIDController.getDParam());
+}
+
+void ActuatorController::updatePIDControllerI(const float newI, const bool isVelocityPID) {
+  // Change I, but keep P and D the same
+  isVelocityPID ? m_velocityPIDController.setTunings(m_velocityPIDController.getPParam(), newI,
+                                                     m_velocityPIDController.getDParam())
+                : m_positionPIDController.setTunings(m_positionPIDController.getPParam(), newI,
+                                                     m_positionPIDController.getDParam());
+}
+
+void ActuatorController::updatePIDControllerD(const float newD, const bool isVelocityPID) {
+  // Change D, but keep I and P the same
+  isVelocityPID ? m_velocityPIDController.setTunings(m_velocityPIDController.getPParam(),
+                                                     m_velocityPIDController.getIParam(), newD)
+                : m_positionPIDController.setTunings(m_positionPIDController.getPParam(),
+                                                     m_positionPIDController.getIParam(), newD);
+}
+
+void ActuatorController::updatePIDControllerBias(const float newBias, const bool isVelocityPID) {
+  isVelocityPID ? m_velocityPIDController.setBias(newBias) : m_positionPIDController.setBias(newBias);
+}
+
+void ActuatorController::updatePIDControllerDeadzone(const float newDeadzone, const bool isVelocityPID) {
+  isVelocityPID ? m_velocityPIDController.setDeadZoneError(newDeadzone)
+                : m_positionPIDController.setDeadZoneError(newDeadzone);
+}
+
+bool ActuatorController::updatePIDP(const float newP, const bool isVelocityPID) {
+  if (!m_allowPIDTuningMutex.trylock_for(200ms)) return false;
```

Comment:
Is there any benefit to having this delay be hard coded as opposed to being variable?